packages/commonwealth/client/scripts/views/modals/AuthModal/EmailForm/EmailForm.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import CWLoadingSpinner from 'views/components/component_kit/new_designs/CWLoadingSpinner';
+import { CWForm } from '../../../components/component_kit/new_designs/CWForm';
+import { CWTextInput } from '../../../components/component_kit/new_designs/CWTextInput';
+import { CWButton } from '../../../components/component_kit/new_designs/cw_button';
+import './EmailForm.scss';
+import { emailValidationSchema } from './validation';
+
+type EmailFormProps = {
+  onCancel: () => any;
+  onSubmit: (values: { email: string }) => any;
+  isLoading?: boolean;
+};
+
+const EmailForm = ({ onSubmit, onCancel, isLoading }: EmailFormProps) => {
+  return (
+    <CWForm
+      className="EmailForm"
+      validationSchema={emailValidationSchema}
+      onSubmit={!isLoading ? onSubmit : () => {}}
+    >
+      <CWTextInput
+        fullWidth
+        hookToForm
+        name="email"
+        label="Email address"
+        placeholder="Email address"
+      />
+
+      {isLoading && <CWLoadingSpinner />}
```

Comment:
should we show this along with other content or instead of the content?

Reply:
I don't have confirmation from design/product (this was added ad-hoc), but I think it looks good with the content. We would need a constant pattern here for similar forms. Will ask it from the team and update.